### DISCUSSION TITLE
What are the main reasons for different RUE in APSIM-Wheat and other models?

### DISCUSSION THREAD

**Discussion by BrianCollinss** [APSIMInitiative/ApsimX#9245] [Q&A] [answered]
One reason is that APSIM does not use PAR, instead intercepted radiation (by green leaves). But, this cannot explain the substantial difference between APSIM-Wheat (1.5 g/MJ), CERES-Wheat and SIMPLACE. And some field studies reported RUEs of up to 3.6 g/MJ.

RUE = 3.05 g/MJ PAR --> Ritchie, J.T., Godwin, D.C., Otter, S., 1984. CERES-Wheat: A user oriented wheat yield model. Preliminary documentation. AGRISTARS Publication No. YM-U3-04442-JSC-18892, East Lansing, Michigan, p. 252.

RUE = 2.7 g/MJ PAR --> Stella, T., Webber, H., Rezaei, E. E., Asseng, S., Martre, P., Dueri, S., Guarin, J. R., Pequeno, D. N. L., Calderini, D. F., Reynolds, M., Molero, G., Miralles, D., Garcia, G., Slafer, G., Giunta, F., Kim, Y. U., Wang, C., Ruane, A. C., & Ewert, F. (2023). Wheat crop traits conferring high yield potential may also improve yield stability under climate change. In Silico Plants, 5(2), 1–16.

**Comment by BrianCollinss** [comment]
@HamishBrownPFR any chance I can pick your brain here?

**Comment by HamishBrownPFR** [comment, marked as answer]
I think the difference between PAR and total radiation does explain the difference.  APSIM uses the product total incoming radiation, and the fraction intercepted by the canopy to calculate biomass production with a RUE of 1.5 g/MJ.  Radiation in the Photosynthetically Active spectrum makes up about half of total incoming radiation.  So, if APSIM were using PAR instead of total, we would have to double (approx) RUE to 3.0 g/MJ to achieve the same predictions of biomass production.  This would put APSIM in the same ballpark as the others.

**Reply by BrianCollinss** [reply]
Thanks @HamishBrownPFR.

**Comment by hut104** [comment]
And RUE is for above and below ground growth.
